clustering/k-means-clustering-plugin/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "polus-plugins-clustering-k-means"
+version = "0.3.2"
+description = ""
+authors = [
+"Jayapriya Nagarajan <jayapriya.nagarajan@nih.gov>",
+"Kelechi Nina Mezu <nina.mezu@nih.gov>",
+"hamshkhawar <hamdahshafqat.abbasi@nih.gov>"
+]
+readme = "README.md"
+packages = [{include = "polus", from = "src"}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+filepattern = "^2.0.0"
+typer = "^0.7.0"
+nyxus = "^0.5.0"
+fastapi = "^0.92.0"
+astropy = "5.2.1"
+vaex = "^4.7.0"
+scikit_learn="^1.0.2"
```

Comment:
Why does this package depend on nyxus, fastapi, and astropy? I think these can be cut out of here. I think astropy is a dependency of vaex, so removing it shouldn't affect the plugin.